src/CommonLib/Enums/CollectionMethods.cs

```diff
@@ -24,6 +24,7 @@ public enum ResolvedCollectionMethod
         UserRights = 1 << 15,
         CARegistry = 1 << 16,
         DCRegistry = 1 << 17,
+        CertServices = 1 << 18,
         LocalGroups = DCOM | RDP | LocalAdmin | PSRemote,
         ComputerOnly = LocalGroups | Session | UserRights | CARegistry | DCRegistry,
         DCOnly = ACL | Container | Group | ObjectProps | Trusts | GPOLocalGroup,
```

Comment:
I think it should be added to DCOnly as it is only collecting from DC. Would add it to Default as well.

Reply:
I have pushed up these changes 👍